[dev/latest-guides-auth-request] Fixed grammatical error (#431)
strapi/documentation: docs/developer-docs/latest/guides/auth-request.md

## Login as a Reader

> To login as a user, you will need to follow this example: [login documentation](/developer-docs/latest/development/plugins/users-permissions.md#login).

Reviewer: ```suggestion
To login as a user, you will need to follow the [login documentation](/developer-docs/latest/development/plugins/users-permissions.md#login).
```
